src/mono/wasm/runtime/driver.c

```diff
@@ -685,23 +685,27 @@ mono_wasm_invoke_method_ref (MonoMethod *method, MonoObject **this_arg_in, void
 	MONO_EXIT_GC_UNSAFE;
 }
 
-EMSCRIPTEN_KEEPALIVE MonoObject*
-mono_wasm_invoke_method_bound (MonoMethod *method, void* args)// JSMarshalerArguments
+EMSCRIPTEN_KEEPALIVE int
+mono_wasm_invoke_method_bound (MonoMethod *method, void* args /*JSMarshalerArguments*/, MonoObject **_out_exc)
```

Comment:
If the ts side type is MonoStringRef, this needs to also be `MonoString **`. Otherwise it's possible something that isn't a MonoString can end up in it.